HybridCar/src/test/java/Controller/CarBussinesTest.java

```diff
@@ -0,0 +1,31 @@
+package Controller;
+
+import Model.Vehicle;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class CarBussinesTest {
+   public CarBussines carBussines = new CarBussines();
+
+    @Before
+    public void init(){
+    CarBussines carBussines = new CarBussines();
+    List<Vehicle> vehicles = new ArrayList();
+    vehicles = carBussines.addVehicle();
+   
+}
+    @Test
+    public void cehckList(){
+
+    int nrCars = carBussines.filterList();
```

Comment:
what do you test here? How do you know that you will get 1 car? There is no parameter setup. This is just one test. Do you think that this ensures the quality of your code?